ebusd-2.1.x/en/vaillant/bai.308523.inc

```diff
@@ -20,6 +20,8 @@ r,,WP,d.10 Central heating pump,,,,"4400",,,onoff,,,internal heating pump
 r,,extWP,d.11 external pump,,,,"3F00",,,onoff,,,External heating pump
 r,,Storageloadpump,d.12 storage load pump,,,,"9E00",,,percent0,,,tank load pump demand
 r,,CirPump,d.13 Circulation pump,,,,"7B00",,,onoff,,,Hot water circulation pump (via external module)
+r;wi,,PumpPowerDesired,d.14 Desired heating pump power,,,,A100,,,UCH,0=auto;1=53;2=60;3=70;4=85;5=100,%,PWM-Desired central heating pump power,,,
```

Comment:
if this is supposed to be added (instead of just linking the right product [as described here](https://github.com/john30/ebusd-configuration/issues/208#issuecomment-782590416)), then please also adjust the "de" file

Reply:
scan.08  = Vaillant;BAI00;0104;7803
scan.08 id = 21;19;44;0010021887;0001;012451;N0
scan.15  = Vaillant;70000;0613;6903
scan.15 id = 21;19;43;0020171316;0082;026204;N4